lib/paged_vertical_calendar.dart

```diff
@@ -101,6 +102,10 @@ class PagedVerticalCalendar extends StatefulWidget {
   /// Select start day of the week to be Sunday
   final bool startWeekWithSunday;
 
+  /// Hide certain Weekdays eg.Weekends by providing
+  /// `[DateTime.sunday,DateTime.monday]`. defaults to `[]`
```

Comment:
if you explain default functionality explain default functionality consequences:
`by default all weekdays are shown` instead of `[ ]`